### PR TITLE
Edinburgh - PUT or POST interval action not work

### LINKED ISSUE #1497: PUT or POST interval action not work
I don't know how to create POST or PUT interval action and didn't found any clue in the source code.
The http request body is nil and params only use for ContentLengthKey
```
	req, err := http.NewRequest(httpMethod, executingUrl, nil)
	req.Header.Set(ContentTypeKey, ContentTypeJsonValue)

	params := strings.TrimSpace(intervalAction.Parameters)

	if len(params) > 0 {
			req.Header.Set(ContentLengthKey, string(len(params)))
	}
```
https://github.com/edgexfoundry/edgex-go/blob/master/internal/support/scheduler/schedule.go#L465

### Affected Services
support-scheduler

### Is this a regression?
I not sure.  I didn't found any description to setup a POST or PUT action.
https://docs.edgexfoundry.org/Ch-APISupportingServicesScheduling.html

## 🌍  Your Environment
**Deployment Environment:**
macos
**EdgeX Version:**
edgex-go master branch

### PR DESCRIPTION
**Pull request by xmlviking**
Fix #1497
Fixed issue with Scheduler POST and PUT HTTP Verbs not supplying JSON body when creating interval actions.
Fixed scheduler rst documentation around the use of interval and interval actions.

Signed-off-by: xmlviking <ecotter@gmail.com>

### CODE REVIEW

**Comment by xmlviking** [comment]
recheck

**Comment by xmlviking** [comment]
recheck

File: docs/api/Ch-APISupportingServicesScheduling.rst [hunk 1]
```diff
@@ -26,93 +26,75 @@ https://github.com/edgexfoundry/edgex-go/blob/master/api/raml/support-scheduling
...
+    "target": "core-data",
+    "protocol": "http",
+    "httpMethod": "DELETE",
+    "address": "localhost",
+    "port": 48080,
+    "path": "/api/v1/event/scrub"}' "http://localhost:48085/api/v1/intervalaction"
 
-For testing update the midnight service to be every 60 seconds
 
+An interval action which connects to edgex-device-modbus every 20 seconds and updates a random flag. Using the every20s interval
```
Comment (weichou1229) [inline docs/api/Ch-APISupportingServicesScheduling.rst]: This is should be edgex-device-virtual.

File: docs/api/Ch-APISupportingServicesScheduling.rst [hunk 2]
```diff
@@ -26,93 +26,75 @@ https://github.com/edgexfoundry/edgex-go/blob/master/api/raml/support-scheduling
...
 ::
 
-   curl -X PUT -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d '{ 
-   "name":"midnight",
-   "frequency":"PT60S"}' "http://localhost:48081/api/v1/schedule"
+curl -X POST -H "Content-Type: application/json" -d '{
+    "name": "put-action",
+    "interval": "every20s",
+    "target": "edgex-device-modbus",
```
Comment (weichou1229) [inline docs/api/Ch-APISupportingServicesScheduling.rst]: This is should be edgex-device-virtual.

File: internal/support/scheduler/schedule.go [hunk 3]
```diff
@@ -498,6 +489,29 @@ func execute(context *IntervalContext, wg *sync.WaitGroup) error {
 	return nil
 }
 
+// EFC We may need to modify this for authorization type in the future
+func getRequestMethod(httpMethod string, executingUrl string, intervalAction contract.IntervalAction) (*http.Request, error) {
```
Comment (weichou1229) [inline internal/support/scheduler/schedule.go]: Why not use getHttpRequest instead of the getRequestMethod?

**Comment by weichou1229** [comment]
@xmlviking Is it duplicate with https://github.com/edgexfoundry/edgex-go/pull/1538 ?

**Comment by xmlviking** [comment]
Different branches.  One is master.  One is Edinburgh

File: docs/api/Ch-APISupportingServicesScheduling.rst [hunk 4]
```diff
@@ -12,7 +12,7 @@ For a description of the architecture, see :doc:`../Ch-Scheduling`
 Introduction
 ============
 
-The following API RESTful Web Services for EdgeX Foundry at this time shows the older development name of "fuse."  The EdgeX Foundry name in the software will be updated soon to "edgexfoundry." 
+The following API RESTful Web Service(s) for EdgeX Foundry."
```
Comment (jdharms) [inline docs/api/Ch-APISupportingServicesScheduling.rst]: Missing something here?

Comment (jdharms) [reply docs/api/Ch-APISupportingServicesScheduling.rst]: This sentence is missing a verb.

Comment (xmlviking) [reply docs/api/Ch-APISupportingServicesScheduling.rst]: LOL :P Updating

**Comment by tsconn23** [comment]
@xmlviking Be advised the docs are being deleted with this PR https://github.com/edgexfoundry/edgex-go/pull/1563

The documentation mods will need to be migrated to the `edinburgh` branch of edgex-docs

**Pull request merged**